### PR TITLE
Update streaming API docs

### PR DESCRIPTION
**Pull request by pko-trackunit**

### CODE REVIEW

File: api-docs/machine-insights-streaming-dataformat.md [hunk 1]
```diff
@@ -0,0 +1,305 @@
...
+| Field              | Example          | Description                                                          |
+|:-------------------|:-----------------|:---------------------------------------------------------------------|
+| Country (optional) | "Denmark"        | Country of the location.                                             |
+| Zipcode (optional) | "9000"           | Zipcode of the location.                                             |
+| City (optional)    | "Aalborg"        | City of the location.                                                |
+| Address (optional) | "Gasværksvej 24" | Street address of the location.                                      |
+| GeoHash (optional) | "u4phd376qbg5"   | [GeoHash](https://en.wikipedia.org/wiki/Geohash) of the location.    |
+| datetime           | 1604042795000    | Unix timestamp in milliseconds, for when this location was recorded. |
+
+## AccessControlKeyUsage
```
Comment (pko-trackunit) [inline api-docs/machine-insights-streaming-dataformat.md:74]: @ext-kkj FYI, copied from old docs site.

File: api-docs/machine-insights-streaming-dataformat.md [hunk 2]
```diff
@@ -0,0 +1,305 @@
...
+    "left",
+    "left",
+    "left"
+  ]
+}
+[/block]
+
+## Hours: CumulativeOperatingHours, CumulativeIdleHours, ...
+
+In general, the field CumulativeOperatingHours is the most interesting: this is the total amount of hours for the machine. Other fields might be present as well, if those datapoints are available for this machine, such as CumulativeIdleHours, CumulativeIdleNonOperatingHours, CumulativeProductiveHours, CumulativeMovingHours, etc. The general format of the Hours fields is:
```
Comment (pwd-trackunit) [inline api-docs/machine-insights-streaming-dataformat.md:119]: How about something more formal? 

The Cumulative Operating Hours field is a crucial parameter that represent total amount of hours that machine was operating. Depending on machine type, also fields below can be present:

- CumulativeIdleHours
- ...

File: api-docs/machine-insights-streaming-dataformat.md [hunk 3]
```diff
@@ -0,0 +1,305 @@
...
+"left"
+]
+}
+[/block]
+
+For the exact semantics and availability of these datapoints, please see the [data model description.](https://dev.trackunit.com/docs/trackunit-data-model)
+
+## CANMessages
+
+> 📘 This is probably not the data you are looking for
```
Comment (pwd-trackunit) [inline api-docs/machine-insights-streaming-dataformat.md:160]: Not sure about that :)

File: api-docs/machine-insights-streaming-dataformat.md [hunk 4]
```diff
@@ -0,0 +1,305 @@
...
+category: 62fbabf8d9095e057cc1cd2c
+parentDocSlug: streaming-api
+---
+The Machine Insights based streaming dataformat provides high-level semantic information about machines. This means that e.g. information about "Operating Hours", "Fuel Remaining in Tank", "DEF Level" and "Battery Voltage" are immediately present in a normalized format - no matter the input source. This is perfectly tailored towards usage in a datalake or in AI-based analytics.
+
+> 📘 All datafields are optional
+>
+> For the same machine some might be filled out in one message while others are filled out in the next message; which fields are filled out depends on the technical details of the datacollection. The interpretation should always be that the individual datapoint was provided at that individual timestamp.
+
+Data is in general in-order, that is most recent datapoint last, but this is not guaranteed. If the most recent data is needed the timestamps should be compared. Any processing of historic data should be prepared to receive and handle data out of order.
```
Comment (pwd-trackunit) [inline api-docs/machine-insights-streaming-dataformat.md:12]: How about: "The data is generally ordered in sequence, with the most recent datapoint appearing last, although this order is not guaranteed." ?

**Review by pwd-trackunit — APPROVED** [review]

File: api-docs/machine-insights-streaming-dataformat.md (hunk 3 above)
Comment (pko-trackunit) [reply api-docs/machine-insights-streaming-dataformat.md:160]: That's just copy-paste from what we already have published on: https://developers.trackunit.com/reference/machine-insights-streaming-dataformat 😜 

I will look at your suggestions later. I want to publish new insights before Bartek sends the email.

File: api-docs/machine-insights-streaming-dataformat.md [hunk 5]
```diff
@@ -0,0 +1,305 @@
+---
+title: Machine Insights Streaming Dataformat
+category: 62fbabf8d9095e057cc1cd2c
+parentDocSlug: streaming-api
+---
+The Machine Insights based streaming dataformat provides high-level semantic information about machines. This means that e.g. information about "Operating Hours", "Fuel Remaining in Tank", "DEF Level" and "Battery Voltage" are immediately present in a normalized format - no matter the input source. This is perfectly tailored towards usage in a datalake or in AI-based analytics.
+
+> 📘 All datafields are optional
+>
+> For the same machine some might be filled out in one message while others are filled out in the next message; which fields are filled out depends on the technical details of the datacollection. The interpretation should always be that the individual datapoint was provided at that individual timestamp.
```
Comment (ujo-trackunit) [inline api-docs/machine-insights-streaming-dataformat.md:10]: It is "data<space>collection"?

File: api-docs/machine-insights-streaming-dataformat.md [hunk 6]
```diff
@@ -0,0 +1,305 @@
...
+| datetime                 | 1604042795000 | [Unix timestamp](https://en.wikipedia.org/wiki/Unix_time) in milliseconds, for when this location was recorded. |
+
+## LocationMetadata
+
+Contains additional information about the Location.
+
+| Field                     | Example | Description                                                                                                                     |
+|:--------------------------|:--------|:--------------------------------------------------------------------------------------------------------------------------------|
+| AccuracyRadius (optional) | 10.0    | Accuracy of the location defined as a radius in metres. Corresponds to the Estimated Horizontal Position Error (EHPE) from GPS. |
+| HighAccuracy (optional)   | true    | The location in com.aemp20.Location has high accuracy                                                                           |
```
Comment (ujo-trackunit) [inline api-docs/machine-insights-streaming-dataformat.md:59]: When is it false?

File: api-docs/machine-insights-streaming-dataformat.md [hunk 7]
```diff
@@ -0,0 +1,305 @@
...
+Contains the approximate street address of the given Location.
+
+| Field              | Example          | Description                                                          |
+|:-------------------|:-----------------|:---------------------------------------------------------------------|
+| Country (optional) | "Denmark"        | Country of the location.                                             |
+| Zipcode (optional) | "9000"           | Zipcode of the location.                                             |
+| City (optional)    | "Aalborg"        | City of the location.                                                |
+| Address (optional) | "Gasværksvej 24" | Street address of the location.                                      |
+| GeoHash (optional) | "u4phd376qbg5"   | [GeoHash](https://en.wikipedia.org/wiki/Geohash) of the location.    |
+| datetime           | 1604042795000    | Unix timestamp in milliseconds, for when this location was recorded. |
```
Comment (ujo-trackunit) [inline api-docs/machine-insights-streaming-dataformat.md:72]: Other places has the URL ref. too...

File: api-docs/streaming-api.md [hunk 8]
```diff
@@ -0,0 +1,41 @@
...
+Only the data related to the customer's machines will be made available for the customer.
+
+Data is available in the Streaming API with low latency after it enters the Trackunit Iris platform. There can be delays between the telematic devices (or 3rd party integrations such as ISO feeds) and the Iris platform due to network connectivity or similar. The common case is however that data is available with near-realtime delays.
+
+## Data Formats and Compatibility
+
+Data is available in two formats:
+
+- **Machine Insights-based**: High-level semantic information about machines, normalized and cleansed. Immediately ready to ingest into a data-lake, or for use in AI-based analytics.
+- **RAW data**: As a direct replacement for the classic Unit APIs, this format is a direct migration path from those.
```
Comment (ujo-trackunit) [inline api-docs/streaming-api.md:23]: Perhaps we should mentioned something similar as with CANMessages vs. MI - that MI stream should be preferred to RAW?

Comment (ujo-trackunit) [reply api-docs/streaming-api.md:23]: Is there any downside for the customer to have both streams?

File: api-docs/streaming-api.md [hunk 9]
```diff
@@ -0,0 +1,41 @@
...
+
+We recommend the Machine Insights based topic for any future integrations.
+
+Technically, the data is provided in the [binary Avro dataformat](http://avro.apache.org/) which is very efficient, while at the same time allowing us to extend the dataformat in the future with additional datapoints. A Schema Registry is used to allow producers and consumers to evolve while maintaining compatibility. We will only do compatible changes to the dataformats and stay compatible with the previous version of the schemas, and announce any changes. We recommend to stay up to date, and implement these schema changes in a timely fashion in order to have access to all the available datapoints.
```
Comment (ujo-trackunit) [inline api-docs/streaming-api.md:27]: Is it "data<space>formats"? Perhaps we should have the chatbot check it?

File: api-docs/streaming-api.md (hunk 8 above)
Comment (pko-trackunit) [reply api-docs/streaming-api.md:23]: > Perhaps we should mentioned something similar as with CANMessages vs. MI - that MI stream should be preferred to RAW?

In the next paragraph there's "We recommend the Machine Insights based topic for any future integrations." 

> Is there any downside for the customer to have both streams?

I don't know. Is RAW data published in separate topic that has to be configured first? I guess that's slightly more work for us.
I guess it's additional work for customer too - they would have to consume both topics and match the data on their own.

**Pull request merged**